src/views/Home/index.tsx

```diff
@@ -47,8 +47,10 @@ const Home = () => {
             {message}
 
             <Grid>
-              <PageHeading>{t('title')}</PageHeading>
-              <p className="line-height-body-5 font-body-lg text-light margin-bottom-6">
+              <PageHeading className="margin-bottom-1">
+                {t('title')}
+              </PageHeading>
+              <p className="line-height-body-5 font-body-lg text-light margin-top-0 margin-bottom-6">
```

Comment:
I think this should probably have a `margin-bottom-3`.  The `margin-bottom-6` is a bit more space than defined in the ticket screenshot